examples/reading_logs_via_rule_message/reading_logs_via_rule_message.h

```diff
@@ -103,7 +103,6 @@ static void *process_request(void *data) {
     }
 
     pthread_exit(NULL);
```

Comment:
To avoid [Sonarcloud's](https://sonarcloud.io/project/issues?id=owasp-modsecurity_ModSecurity&open=AY1CfKFIrsSpWCKX0w9K) next warning, you should replace `NULL` literal here by `nullptr`.

Reply:
Thank you. Done.